docs/sphinx/using/backends/hardware/superconducting.rst

```diff
@@ -157,39 +172,33 @@ Submitting
         .. code:: python
 
             cudaq.sample(kernel, shots_count=10000)
-  
+
+        To see a complete example for using IQM server backends, take a look at our :doc:`Python examples<../../examples/examples>`.
+
+
 .. tab:: C++
-    
-        To target quantum kernel code for execution on an IQM Server,
-        pass the ``--target iqm`` flag to the ``nvq++`` compiler, along with a specified ``--iqm-machine``.
 
-        .. note::
-            The ``--iqm-machine`` is  a mandatory argument. This provided architecture must match
-            the device architecture that the program has been compiled against. The hardware architecture for a
-            specific IQM Server may be checked  via `https://<IQM server>/cocos/quantum-architecture`.
+        To target quantum kernel code for execution on an IQM Server,
+        pass the ``--target iqm`` flag to the ``nvq++`` compiler.
 
         .. code:: bash
 
-            nvq++ --target iqm --iqm-machine Crystal_5 src.cpp
+            nvq++ --target iqm src.cpp
 
-        Once the binary for a specific IQM QPU architecture is compiled, it can be executed against any IQM Server with the same QPU architecture:
+        Once the binary for an IQM QPU is compiled, it can be executed against any IQM Server:
 
         .. code:: bash
 
-            nvq++ --target iqm --iqm-machine Crystal_5 src.cpp -o program
-            IQM_SERVER_URL="https://demo.qc.iqm.fi/cocos" ./program
-
-            # Executing the same program against an IQM Server with a different underlying QPU
-            # architecture will result in an error.
-            IQM_SERVER_URL="https://<Crystal_20 IQM Server>/cocos" ./program
+            nvq++ --target iqm src.cpp -o program
+            IQM_SERVER_URL="https://demo.qc.iqm.fi/" ./program
 
         To emulate the IQM machine locally, without submitting to the IQM Server,
         you can also pass the ``--emulate`` flag to ``nvq++``. This will emit any target
         specific compiler diagnostics, before running a noise free emulation.
 
         .. code:: bash
 
-            nvq++ --emulate --target iqm --iqm-machine Crystal_5 src.cpp
+            nvq++ --emulate --target iqm src.cpp
```

Comment:
I believe emulation would not work fully as expected - it would not do include the layout for a specific device. 
Let me get back on that with some options.